process_email.py

```diff
@@ -686,7 +687,7 @@ def write_with_new_filename(self, data, dict_to_fill, file_name, as_byte=False):
             if hasattr(Vault, 'get_vault_tmp_dir'):
                 fd, full_path = tempfile.mkstemp(dir=Vault.get_vault_tmp_dir())
             else:
-                fd, full_path = tempfile.mkstemp(dir='/opt/phantom/vault/tmp')
+                fd, full_path = tempfile.mkstemp(dir=os.path.join(paths.PHANTOM_VAULT, "tmp"))
```

Comment:
This can be further improved, have a look here https://github.com/splunk-soar-connectors/rss/pull/14/files#r1410307392

Reply:
updated changes